### ISSUE TITLE
react16 import prop-types

### ISSUE THREAD

**Issue by headwinds** [danielarias123/react-multi-toggle#4]
Since react.PropTypes is deprecated in react16 would you consider importing it via the npm package?

I see you've already done some work with that index es6 version towards this goal

I just tried to upgrade from react15 to react16 and I'm getting an error in my build with webpack and babel - I'll share a snippet from my console logs:

It complains that 
```
Uncaught TypeError: Cannot read property 'any' of undefined
    at Object.<anonymous> (index.js:98)
``` 

when I click on the red link, I can see culprit:

```
MultiToggle.propTypes = {
  className: _react.PropTypes.any,
  options: _react.PropTypes.array.isRequired,
  selectedOption: _react.PropTypes.any.isRequired,
  onSelectOption: _react.PropTypes.func,
  label: _react.PropTypes.any
};
```

So I'm thinking using the [prop-types](https://github.com/facebook/prop-types#prop-types) library instead may resolve this issue react16 create-react-app builds...

In the meantime, simply deleting the MultiToggle.propTypes  section from index.js fixed the problem for the time being for react16

**Comment by peisenmann** [comment]
@danielarias123 What are the chances I could trouble you to publish this to npm? 🥇

**Comment by danielarias123** [comment]
@peisenmann yup just published version 1.1.0. Thanks again!
